CHANGELOG.asciidoc

```diff
@@ -64,6 +64,10 @@ https://github.com/elastic/beats/compare/v6.5.2...6.5[Check the HEAD diff]
 
 *Filebeat*
 
+- Added `detect_null_bytes` selector to detect null bytes from a io.reader. {pull}9210[9210]
```

Comment:
Seems the rebase mess up the backport a bit. Can you clean this up?

Reply:
Lesson learnt. Even when you expect an almost clean rebase, double check changes.